crates/containerd-shim-wasm/src/sandbox/error.rs

```diff
@@ -39,6 +39,7 @@ pub enum Error {
     #[error("{0}")]
     Json(#[from] serde_json::Error),
     /// Error from the system
+    #[cfg(unix)]
```

Comment:
What's the equivalent of errno on the Windows side?

Reply:
windows error codes: https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-

I will add section here for windows when/if needed in the next pr that adds the Windows implementation.